tests/legacy-cli/verdaccio.yaml

```diff
@@ -18,7 +18,7 @@ packages:
     access: $all
     publish: $all
 
-  '@ngtools/*':
+  '@ngtools/webpack':
```

Comment:
Was this change necessary or was it just cleanup? Just wondering, because `@angular-devkit/*` still has the star above.

Reply:
“@ngtools/json-schema” is used by ng-packgr.  Since it no longer exists, it’s not published locally which causes the installation of ng-packgr to fail within the E2E tests.